indico/migrations/versions/20210630_1349_5cbb0eb12eb3_add_last_used_ip_and_use_count_to_tokens.py

```diff
@@ -0,0 +1,33 @@
+"""Add last_used_ip and use_count to tokens
+
+Revision ID: 5cbb0eb12eb3
+Revises: 1b7e98f581bc
+Create Date: 2021-06-30 13:49:55.566636
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '5cbb0eb12eb3'
```

Comment:
Just out of curiosity, is it not better to compact all the migrations into one after you are done with the PR?

Reply:
if they all touched the same tables then yes - but this one also affects the (existing) table with the oauth tokens. So we need a revision for this anyway...

Moving the changes made to the new table into the original revision creating it wouldn't be a bad idea per se. The main reason I didn't do it is that I don't plan to squash the commits of this PR as they are nicely split into different somewhat unrelated parts. Even though the only place where this would be an issue is when someone wanted to git-bisect and also follow the DB changes in the commits (which is very unlikely to happen): When you are on the commit adding these fields and want to downgrade, you'd also have to downgrade the creation of the personal tokens table, even though that revision was introduced in another commit..